app/assets/stylesheets/views/signup-modal.scss

```diff
@@ -83,16 +44,18 @@
       width: 100%;
       margin: 0 auto;
       justify-self: center;
-      padding: 0 var(--su-8) var(--su-7);
+      padding: 0 var(--su-8) var(--su-4);
     }
   }
 
   &__footer {
     border-top: 2px solid var(--base-100);
-    padding: var(--su-2) var(--su-4);
+    padding: var(--su-2) var(--su-4) 0 var(--su-4);
+    margin: 0 calc(-1 * var(--modal-body-padding)) 0
+      calc(-1 * var(--modal-body-padding));
```

Comment:
I don't love this, but it allows the footer horizontal bar to extend beyond the default crayons modal content so we keep the previous look of this modal

Reply:
Maybe we can scrap the current look to make things uniform. Thoughts @ludwiczakpawel and @lisasy ?

Reply:
We definitely should. Signup modal is very one-offy unfortunately :(

I'm happy to take it for a spin, but that should be a separate issue/PR.

Reply:
I'll create an issue for this, and note that this funky css should be removed as part of that process 🙂

Reply:
#12663